src/main/java/frc/robot/subsystems/drive/ModuleIOTalonFX.java

```diff
@@ -217,4 +217,8 @@ public void setTurnBrakeMode(boolean enable) {
     config.NeutralMode = enable ? NeutralModeValue.Brake : NeutralModeValue.Coast;
     turnTalon.getConfigurator().apply(config);
   }
+
+  public double getSelectedSensorVelocity() {
```

Comment:
Who uses this command? This value should be gotten from inputs.driveVelocityRotationsPerSec

Reply:
It's not used. I removed it